docs/javascript/browser-apis-and-jquery.md

```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
 
 var catIntervalID = setInterval(increaseCatLikes, 5000);
 ```
+
```

Comment:
I liked it that you ran prettier on your earlier commit, I'm doing that now too. Let's do that from now on. If you haven't on this commit, do it on the "fixes" commit.